code/modules/unit_tests/reagent_id_typos.dm

```diff
@@ -4,7 +4,6 @@
 
 /datum/unit_test/reagent_id_typos/Run()
 	build_chemical_reactions_lists()
-	build_chemical_reagent_list()
```

Comment:
Unintended removal?

Reply:
No, chemical reagents not being generated right away was causing issues with organs were being created, because organs now must be created immediately (because we want to generate humans right away), which means that they need to be edible right away, which tries to create nutriments, but nutriments weren't created until after asset generation was complete.

Everything relevant here has been made into a GLOBAL_LIST_INIT.